app-shells/starship/starship-1.9.1.ebuild

```
@@ -0,0 +1,354 @@
+	sha-1-0.10.0
+	sha1-0.6.1
+	sha1_smol-1.0.0
+	sha2-0.9.9
+	shadow-rs-0.11.0
+	shared_library-0.1.9
+	shell-words-1.1.0
+	signal-hook-0.1.17
+	signal-hook-0.3.14
+	signal-hook-registry-1.4.0
+	siphasher-0.3.10
+	slab-0.4.6
+	smallvec-1.8.1
+	socket2-0.4.4
+	starship-battery-0.7.9
+	static_assertions-1.1.0
+	strsim-0.10.0
+	strum-0.22.0
+	strum_macros-0.22.0
+	syn-1.0.98
+	systemstat-0.1.11
+	tempfile-3.3.0
+	termcolor-1.1.3
+	terminal_size-0.1.17
+	terminfo-0.7.3
+	termios-0.3.3
+	termtree-0.2.4
+	termwiz-0.15.0
+	textwrap-0.15.0
+	thiserror-1.0.31
+	thiserror-impl-1.0.31
+	time-0.1.44
+	time-0.3.11
+	tinyvec-1.6.0
+	tinyvec_macros-0.1.0
+	toml-0.5.9
+	toml_edit-0.14.4
+	tracing-0.1.35
+	tracing-attributes-0.1.21
+	tracing-core-0.1.28
+	typenum-1.15.0
+	ucd-trie-0.1.3
+	uds_windows-1.0.2
+	unicase-2.6.0
+	unicode-bidi-0.3.8
+	unicode-ident-1.0.1
+	unicode-normalization-0.1.20
+	unicode-segmentation-1.9.0
+	unicode-width-0.1.9
+	unicode-xid-0.2.3
+	uom-0.30.0
+	url-2.2.2
+	urlencoding-2.1.0
+	utf8parse-0.2.0
+	vcpkg-0.2.15
+	version_check-0.9.4
+	versions-4.1.0
+	vtparse-0.6.1
+	waker-fn-1.1.0
+	wasi-0.9.0+wasi-snapshot-preview1
+	wasi-0.10.0+wasi-snapshot-preview1
+	wasi-0.11.0+wasi-snapshot-preview1
+	wepoll-ffi-0.1.2
+	which-4.2.5
+	winapi-0.3.9
+	winapi-i686-pc-windows-gnu-0.4.0
+	winapi-util-0.1.5
+	winapi-x86_64-pc-windows-gnu-0.4.0
+	windows-0.24.0
+	windows-0.38.0
+	windows-sys-0.36.1
+	windows_aarch64_msvc-0.36.1
+	windows_aarch64_msvc-0.38.0
+	windows_i686_gnu-0.24.0
+	windows_i686_gnu-0.36.1
+	windows_i686_gnu-0.38.0
+	windows_i686_msvc-0.24.0
+	windows_i686_msvc-0.36.1
+	windows_i686_msvc-0.38.0
+	windows_x86_64_gnu-0.24.0
+	windows_x86_64_gnu-0.36.1
+	windows_x86_64_gnu-0.38.0
+	windows_x86_64_msvc-0.24.0
+	windows_x86_64_msvc-0.36.1
+	windows_x86_64_msvc-0.38.0
+	winres-0.1.12
+	winrt-notification-0.5.1
+	xml-rs-0.8.4
+	yaml-rust-0.4.5
+	zbus-2.3.2
+	zbus_macros-2.3.2
+	zbus_names-2.1.0
+	zvariant-3.4.1
+	zvariant_derive-3.4.1
+"
+
+inherit cargo
+
+DESCRIPTION="The minimal, blazing-fast, and infinitely customizable prompt for any shell"
+HOMEPAGE="https://starship.rs/"
+SRC_URI="
+	https://github.com/starship/starship/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	$(cargo_crate_uris)
+"
+
+LICENSE="Apache-2.0 BSD BSD-2 CC0-1.0 ISC MIT Unlicense WTFPL-2 ZLIB"
+
+SLOT="0"
+KEYWORDS="~amd64"
+
+BDEPEND=">=virtual/rust-1.60"
+DEPEND=">=dev-libs/libgit2-1.4.0:="
+RDEPEND="${DEPEND}"
+
+QA_FLAGS_IGNORED="usr/bin/starship"
+
+src_configure() {
+	export PKG_CONFIG_ALLOW_CROSS=1
+	export LIBGIT2_SYS_USE_PKG_CONFIG=1
+	export OPENSSL_NO_VENDOR=true
+
+	cargo_src_configure
+}
+
+src_install() {
+	cargo_src_install
+	dodoc README.md CHANGELOG.md
+
+	einstalldocs
+}
+
+pkg_preinst() {
+	UPGRADE_FROM_PRE_1_9_0=0
+
+	if has_version "<${CATEGORY}/${PN}-1.9.0" ; then
+		UPGRADE_FROM_PRE_1_9_0=1
+	fi
+}
+
+pkg_postinst() {
+	if [[ "${UPGRADE_FROM_PRE_1_9_0}" -eq 1 ]] ; then
+		einfo "vicmd_symbol config option was renamed to vimcmd_symbol"
```

Please use a full sentence.
```suggestion
		einfo "Note that vicmd_symbol config option was renamed to vimcmd_symbol in version 1.9"
```